utils/build-presets.ini

```diff
@@ -865,6 +865,7 @@ relocate-xdg-cache-home-under-build-subdir
 [preset: buildbot_linux]
 mixin-preset=mixin_linux_installation
 build-subdir=buildbot_linux
+llvm-targets-to-build=X86;ARM;AArch64;PowerPC;SystemZ;Mips;RISCV
```

Comment:
Isn't RISC-V built by default now?

Reply:
Not the way it was building invoked by the Swift build script.